apps/storybook-react/.storybook/main.ts

```diff
@@ -38,10 +38,6 @@ const config: StorybookConfig = {
           __dirname,
           '../../../node_modules/@metamask/design-tokens',
         ),
-        '@components': path.resolve(
-          __dirname,
-          '../packages/react-components/src',
-        ),
```

Comment:
Storybook seems to be working without this and it was before we agreed on the names of the packages so it's a little out of date